bin/run.sh

```diff
@@ -24,16 +24,22 @@ test_file="${2}/${1}.spec.js"
 result_file="${3}/results.json"
```

Comment:
```suggestion
result_file="${3}results.json"
```

Reply:
Also line 21:

```diff
- test_file="${2}/${1}.spec.js"
+ test_file="${2}${1}.spec.js"
```

And line 16

```diff
- # ./run.sh two-fer path/to/two-fer/solution/folder/ path/to/output-directory/
+ # ./run.sh two-fer path/to/two-fer/solution/folder path/to/output-directory
```

Reply:
@joshgoebel Could you do these and check it all works with now I've made the fix upstream? Thanks :)

Reply:
(Thanks @SleeplessByte)